data/human/hails.txt

```diff
@@ -2082,6 +2082,94 @@ phrase "friendly civilian"
 		"Zug. I used to watch the militia ships fight against the pirate ships in the sky from my porch."
 		"New Holland. It gets too boring there, though, so I fly the stars instead."
 		"Bounty. I bought my first ship when the manufacturing industry started to develop there."
+		
+phrase "pyramid_scheme_company"
+	word
+		"Magellan "
+		"Dehydrated "
+		"Pontificated "
+		"General "
+		"Sacrosanct "
+		"Liberated "
+		"Literal "
+		"Family "
+		"Hyper-real "
+	word
+		"Dynamix"
+		"Unlimited"
+		"Orbital"
+		"Gigalight"
+		"Traders"
+		"Voidcraft"
+		"Whispers"
+
+phrase "friendly civilian"
+	word
+		"Hi! Good to see you, uh, "
+		"Well, if it isn't "
+		"Hi there, uhm, "
+		"I've been trying to get a hold of you, "
+	word
+		"Captain <Last>. "
+	word
+		"I'm living the dream as a CEO of my own company. "
+		"I used to be just like you, but now I'm really somebody! "
+		"Don't you want to be your own boss? "
+		"Everybody is talking about this slick new fad! "
+	word
+		"You could get rich selling for me!"
+		"Get in on the ground floor!"
+		"We're about to go orbital, you should sign up while you still can!"
+	word
+		" "
+	word
+		" "
+	word
+		" "
+	word
+		"Find me "
+		"You can get me "
+		"Call me "
+		"Drop me a line "
+		"Buzz me "
+	word
+		"at ${name}@${pyramid_scheme_company}."
```

Comment:
```suggestion
		"at ${pyramid_scheme_company}. "
	word
		"Ask for ${name}."
		"Tell them ${name} sent you."
		"Mention my name, ${name}, for a discount on your first order!"
```

Reply:
Remember to hit the commit suggestions button : )